server/src/main/java/org/elasticsearch/action/admin/cluster/reroute/ClusterRerouteResponse.java

```diff
@@ -45,7 +44,6 @@ public class ClusterRerouteResponse extends ActionResponse implements IsAcknowle
     /**
      * To be removed when REST compatibility with {@link org.elasticsearch.Version#V_8_6_0} / {@link RestApiVersion#V_8} no longer needed
      */
-    @UpdateForV9(owner = UpdateForV9.Owner.DISTRIBUTED_COORDINATION)    // to remove from the v9 API only
     @UpdateForV10(owner = UpdateForV10.Owner.DISTRIBUTED_COORDINATION)  // to remove entirely
```

Comment:
I do not think this pr closes https://github.com/elastic/elasticsearch/issues/88978, until this is removed in 10, is it?

Reply:
I think it does - at least from the users' perspective with this change we no longer return the cluster state as described in #88978. There's some tech debt left behind (because of the need to support the v8 API still) but that's tracked with the `@UpdateForV10` annotation, we don't need to keep an issue open too.